Incomplete Project.

cdunn95 opened the pull request:
This project is currently incomplete. The title and body of this post will be edited upon completion.

Commit:
Basic foundation of the app including UI and MVC classes.

ilqarilyasov commented on Reading List/Reading List/BookController.swift:
```diff
@@ -0,0 +1,73 @@
+    }
+    func delete(bookToDelete book: Book){
+        let index = books.index(of: book)
+        books.remove(at: index!)
```
I would suggest you try to avoid force unwrapping, it's not a good practice. In fact optional Swift's one of main differences from other languages, because you're checking values to avoid crashes. Check [this](https://blog.timac.org/2017/0628-swift-banning-force-unwrapping-optionals/) out 
```suggestion
        guard let index = books.index(of: book) else { return }
        books.remove(at: index)
```

ilqarilyasov reviewed (commented):
Hey Cameron, you've implemented lots of functions in `BookController`, you got the model object right but the rest is empty.  App has only a table view. Couldn't add a book. Try to complete it and let me know. And don't forget to make more commits

cdunn95 committed:
Fixed some errors in the ReadMe

Fixed numbering in part 2 of the instructions and cleared up where to place the function in part 3 step 5.

cdunn2013 commented:
@armadsen

armadsen reviewed (changes requested):
This PR has some good README fixes, but you've made these changes on the same branch as your submission of the project as a student, so I can't merge it. Please pull the README fixes commit onto a fresh branch and submit a new PR.
